lib/fizzy/execute.cpp

```diff
@@ -408,7 +440,7 @@ inline T fmin(T a, T b) noexcept
     if (std::isnan(a) || std::isnan(b))
         return std::numeric_limits<T>::quiet_NaN();  // Positive canonical NaN.
 
-    if (a == 0 && b == 0 && (std::signbit(a) == 1 || std::signbit(b) == 1))
+    if (a == 0 && b == 0 && (signbit(a) == true || signbit(b) == true))
```

Comment:
Are you using C macro now? It does not return a bool. Besides, it should be
```suggestion
    if (a == 0 && b == 0 && (signbit(a) || signbit(b)))
```

Reply:
> Are you using C macro now?

It is a template.